packages/core/src/record.ts

```diff
@@ -187,24 +187,22 @@ class Record<
     return obj
   }
 
-  get<K extends Key>(key: K): Entries[K]
-  get (key: keyof FieldLookup | number): any
-
+  get <K extends keyof Entries = keyof Entries> (key: K): Entries[K]
+  get (n: number): any
   /**
    * Get a value from this record, either by index or by field key.
    *
    * @param {string|Number} key Field key, or the index of the field.
    * @returns {*}
    */
-  get (key: string | number): any {
-    let index
+  get <K extends keyof Entries = keyof Entries> (key: K | number): any {
+    let index: number
     if (!(typeof key === 'number')) {
+      // @ts-expect-error
```

Comment:
I tried to undertand why this is needed.

Sure, I see that `K extends keyof Entries` does not fit `string` which `this._fieldLookup[_]` expects, but what I mean is that I couldn't track down why `keyof Entries` couldn't be typed to be of type `string`. What else could there be in the Record?

Reply:
The _fieldLookup type is defined in the constructor and it doesn't deriver from the `Entries`. I can fix it.